foodnet/membership/models/department.py

```diff
@@ -11,3 +11,9 @@ class Department(models.Model):
 
     def __str__(self):
         return self.name
+
+
+class DepartmentAdministrator(models.Model):
+    department = models.ForeignKey('membership.Department')
+    admin = models.ForeignKey('membership.UserProfile')
```

Comment:
Could this maybe be called `user` or `profile`?
What is the reasoning behind relating to `membership.UserProfile` and not `auth.User`?